doc/api/http.markdown

```diff
@@ -43,12 +43,6 @@ list like the following:
       'Host', 'mysite.com',
       'accepT', '*/*' ]
 
-## http.METHODS
-
-* {Array}
-
-A list of the HTTP methods that are supported by the parser.
-
```

Comment:
Can we expose a way for user-land to maybe add verbs they explicitly want to support? For all verbs that aren't hard coded (i.e. common verbs) I'd want them to just fail as bad input. If someone has a specific need for an unsupported verb they can make a PR or if we have a way to add new verbs in user-land, use that. We (at Grooveshark) get crazy verb requests all the time (for instance thousands of LOAD requests per day) and drop them since they're not defined anywhere and we have no idea how we'd even respond.